plugin.video.retrospect/channels/channel.streams/radio538/chn_radio538.py

```diff
@@ -203,112 +281,154 @@ def create_live_channel(self, result_set):
+                item.isLive = True
+                items.append(item)
 
-        :param dict[str,Any] result_set: The result_set of the self.episodeItemRegex
-
-        :return: A new MediaItem of type 'video' or 'audio' (despite the method's name).
-        :rtype: MediaItem|None
+            else:
+                item = MediaItem(
+                    title, url, media_type=mediatype.AUDIO)
+                items.append(item)
 
-        """
+            Logger.debug("Found stream for %s: %s (%s)", title, url, source)
 
-        Logger.trace(result_set)
+        return items
 
-        start_date = result_set['start']  # 2017-01-01T00:00:00+01:00
-        start_time_stamp = DateHelper.get_date_from_string(start_date.split("+")[0], "%Y-%m-%dT%H:%M:%S")
-        end_date = result_set['end']
-        end_time_stamp = DateHelper.get_date_from_string(end_date.split("+")[0], "%Y-%m-%dT%H:%M:%S")
-        title = "%02d:%02d - %02d:%02d: %s" % (start_time_stamp.tm_hour, start_time_stamp.tm_min,
-                                               end_time_stamp.tm_hour, end_time_stamp.tm_min,
-                                               result_set['title'])
-        item = MediaItem(title, "", media_type=mediatype.EPISODE)
-        item.description = result_set.get("description")
-
-        if "image" in result_set:
-            if not item.description:
-                item.description = result_set["image"].get("alt", None)
-            item.thumb = "https://static.538.nl/%s" % (result_set["image"]['src'],)
-
-        item.set_date(*start_time_stamp[0:6])
-        item.description = result_set.get('description')
-        if "playbackUrls" in result_set and result_set["playbackUrls"]:
-            title_format = "%%02d:%%02d - %s" % (result_set['title'],)
-            item.complete = True
-            hour = start_time_stamp.tm_hour
-            for stream in result_set["playbackUrls"]:
-                if stream.startswith("//"):
-                    stream = "https:%s" % (stream, )
-                part = item.create_new_empty_media_part()
-                part.Name = title_format % (hour, start_time_stamp.tm_min)
-                part.append_media_stream(stream, 0)
-                hour += 1
-        elif "showUrl" in result_set and result_set["showUrl"]:
-            title_format = "%%02d:%%02d - %s" % (result_set['title'],)
-            stream = result_set["showUrl"]
-            item.complete = True
-            hour = start_time_stamp.tm_hour
-            if stream.startswith("//"):
-                stream = "https:%s" % (stream,)
-            part = item.create_new_empty_media_part()
-            part.Name = title_format % (hour, start_time_stamp.tm_min)
-            part.append_media_stream(stream, 0)
-            hour += 1
-        else:
-            Logger.warning("Found item without streams: %s", item)
-            return None
-        return item
+    # No longer used
```

Comment:
Maybe remove the comment block

Reply:
It's still a TODO for the next version, but I did not want to wait that long.